examples/app/simple_hdf5/main.py

```diff
@@ -1,3 +1,9 @@
+''' A simple chart visualizing hdf5 files.
+
+.. note::
+    This visualization needs the hdf5 library to run.
```

Comment:
```suggestion
    This example needs the hdf5 library to run.
```
(let's use "example" verbiage consistently)